Client/Frontend/LoginManagement/BreachAlertsManager.swift

```diff
@@ -26,9 +26,6 @@ struct BreachRecord: Codable, Equatable, Hashable {
 /// A manager for the user's breached login information, if any.
 final public class BreachAlertsManager {
     static let icon = UIImage(named: "Breached Website")?.withRenderingMode(.alwaysTemplate)
```

Comment:
Since we're working in this area, I think it'd be good to clean up a little bit more. 

We can remove this static declaration and just have `UIImage(named: "Breached Website")?.withRenderingMode(.alwaysTemplate)` where it's needed. We can also add `Breached Website` into ImageIdentifiers.